ui/src/router/edit/EditRouterView.js

```diff
@@ -26,28 +28,36 @@ const EditRouterView = ({ projectId, currentRouter, ...props }) => {
   useEffect(() => {
     if (submissionResponse.isLoaded && !submissionResponse.error) {
       addToast({
-        id: "submit-success-create",
-        title: "Router configuration is updated!",
+        id: withDeployment ? "update-success" : "save-success",
+        title: withDeployment
+          ? "Router configuration is updated!"
+          : "Router configuration is saved (but not deployed)!",
```

Comment:
Not sure if this message in the pop-up dialog is sufficient for the user, but in any case, the user most certainly would've been warned in the confirmation window.